docs/content/development/rest/_index.md

```diff
@@ -10,23 +10,21 @@ card:
 
 You need two HTTP Headers to request CDS API:
 
-- `Session-Token: your-cds-token`
+- `Authorization: Bearer your-cds-token`
 - `X-Requested-With: X-CDS-SDK`
 
 ```bash
 # List CDS Project
-curl -H "Session-Token: your-cds-token" -H "X-Requested-With: X-CDS-SDK" https://your-cds-api/project
+curl -H "Authorization: Bearer your-cds-token" -H "X-Requested-With: X-CDS-SDK" https://your-cds-api/project
 ```
 
 ## About CDS Token
 
-The CDS UI uses a non-persistent token. If you want to play with CDS API, you probably need a persistent token.
+The CDS UI uses a session token. If you want to play with CDS API, you probably need a persistent token.
```

Comment:
in fact, it's not a good example, because the user doesn't get a usable session token by creating a new consumer.

The right example would be to:
1) create a builtin consumer
2) Signin with te builtin consumer signin token
3) Retrieve the session-token
4) Use it to get the list of project
5) Resignin when the session-token expires

I guess that this documentation would be in a "about authentication" page

Reply:
Ok I can make a link to this page https://github.com/ovh/cds/pull/4749/files#diff-77dab085999b14ec080ae4a10aa605dc. Should I add something into this page ? https://github.com/ovh/cds/pull/4749/files#diff-77dab085999b14ec080ae4a10aa605dc